tests/unit/brayns/network/TestEntrypoint.cpp

```diff
@@ -0,0 +1,99 @@
+/* Copyright (c) 2015-2022, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include <doctest/doctest.h>
+
+#include <brayns/network/entrypoint/EntrypointBuilder.h>
+#include <brayns/network/entrypoint/EntrypointFinder.h>
+
+#include "MockEntrypoint.h"
+#include "MockInterface.h"
+#include "MockWebSocket.h"
+
+brayns::RequestMessage mockMessage()
```

Comment:
Class with a static method instead of a free function?

Reply:
Yeah I was the one pushing for that :)